data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -629,6 +630,21 @@ states:
           rule_type: DoesNotContainAtLeastOneOf
         tagged_misconception_id: null
         training_data: []
+      - outcome:
+          dest: Item Selection Multi
+          feedback:
+            content_id: feedback_4
+            html: <p>That's good!, no bad options, but choose all good options!</p>
+          labelled_as_correct: false
+          missing_prerequisite_skill_id: null
+          param_changes: []
+          refresher_exploration_id: null
+        rule_specs:
+        - inputs:
+            x: []
```

Comment:
You probably should have something here.

Reply:
Done, added it